unit05/classes&objects.rst

```diff
@@ -0,0 +1,44 @@
+Unit 5 - Classes and Objects Learning Objectives
+================================================
+
+Goals
+-----
+
+- Understand what a Python **object** represents.
+- Understand the basic components of a **class**.
+- Use classes to create objects.
+- Create well-designed classes.
+
+Objectives
+----------
+
+Students will:
+
+- Understand that objects are collections of **properties** and **methods**.
+- Recognize that *properties* are data, while *methods* define the actions the
+  object can take.
+- Access and modify object properties using dot-notation.
+- Call an object method using dot-notation.
+- Understand the relationship between a **class** and an **object**.
+- Recognize that a *class* provides the blueprint from which new objects are
+  created.
+- Successfully declare a class.
+- Use the ``self`` keyword within a class to reference the object.
+- Use ``__init__`` and ``self`` to define initial properties within a class.
+- Add custom methods to a class using the ``def`` keyword.
+- Define a ``__str__`` method for a given class to describe how
+  to deal with printing an object.
+- Be able to declare a new **instance** of a class object.
+- Use objects as arguments, parameters, or return values of functions.
+
+Advanced Topics
+---------------
+
+The following objectives may be included at the teacher's discretion.
+
+Students will:
+
+- Understand **inheritance**.
+- Know how to create a **child class** (*subclass*) that shares properties
```

Comment:
subclass can be a bolded vocab word here

Reply:
Done.